mne_bids/utils.py

```diff
@@ -546,3 +550,18 @@ def _infer_eeg_placement_scheme(raw):
         placement_scheme = 'based on the extended 10/20 system'
 
     return placement_scheme
+
+
+def _check_file_exists(fname):
+    """Checks whether the specified file exists. If so, raises an OSError.
+
+    Parameters
+    ----------
+    fname : str
+        Path to the file we are checking the existance of.
+
+    """
+    if op.exists(fname):
+        raise OSError(errno.EEXIST, '"%s" already exists. Please set '
+                      'write_mode to "overwrite" or "append".' % fname)
+    return False
```

Comment:
you don't need it to return anything ...

Reply:
cf. above comment; Also didn't originally have anything being returned, but I felt it made more sense considering the name.

Reply:
umm ... there are lots of `_check_xxx` functions in `mne` which don't return anything. See [this one](https://github.com/mne-tools/mne-python/blob/master/mne/utils.py#L2278) for instance.